Business.Queries/JobPositions/GetJobPositionByIdQueryHandler.cs

```diff
@@ -1,18 +1,17 @@
-﻿using System.Linq;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Business.Dtos.JobPositions;
+using Business.Queries.Dtos;
 using CCG.AspNetCore.Business.Interface;
 using DataModel;
 using Microsoft.EntityFrameworkCore;
 
 namespace Business.Queries.JobPositions
 {
-        public class GetJobPositionByIdQuery : IQuery<JobPositionDto>
-        {
-            public int Id { get; set; }
-        }
-    public class GetJobPositionByIdQueryHandler : IQueryHandler<GetJobPositionByIdQuery, JobPositionDto>
+    public class GetJobPositionByIdQueryHandler
```

Comment:
This query handler doesn't implement IQueryHandler. Also, it's named "ById" but doesn't take in an Id parameter. Were the changes to this query handler necessary?